Frameworks/CoreLocation/CLLocationManager.mm

```diff
@@ -536,11 +615,16 @@ - (instancetype)init {
         _authorizing = NO;
         _statusUpdateRequested = NO;
         _periodicLocationUpdateRequested = NO;
+        _periodicHeadingUpdateRequested = NO;
         _location = nullptr;
+        _heading = nullptr;
         // Cache the caller's thread object to use it to call delegates on.
         _callerThread = [NSThread currentThread];
         // Initialize WDGGeolocator.
         _uwpGeolocator = [WDGGeolocator make];
+        _uwpCompass = [WDSCompass getDefault];
+        _headingOrientation = CLDeviceOrientationLandscapeLeft;
+        _headingFilter = 0;
```

Comment:
>_headingFilter = 0 [](start = 8, length = 18)

iOS document says the default value for this should be 1? Is it not compliant with Windows?